cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -220,6 +220,32 @@ TEST(TestGdvFnStubs, TestCastVARCHARFromInt64) {
   EXPECT_FALSE(ctx.has_error());
 }
 
+TEST(TestGdvFnStubs, TestCastVARCHARFromMilliseconds) {
+    gandiva::ExecutionContext ctx;
+    uint64_t ctx_ptr = reinterpret_cast<int64_t>(&ctx);
+    int32_t out_len = 0;
+
+    const char* out_str = gdv_fn_castVARCHAR_date64_int64(ctx_ptr, 1619218598237, 100, &out_len);
```

Comment:
This numbers are not helpful for understanding. can you add helper function to convert then to long. Look at StringToTimestamp in precompiled/time_test.cc